tools/codeGen/util/util.go

```diff
@@ -48,6 +48,12 @@ func TODO() {
 	panic("TODO")
 }
 
+// Version of TODO() indicating that the operation is believed to be unambiguous,
+// but is not yet implemented as code in the spec repository.
+func IMPL_FINISH() {
```

Comment:
When I first saw it I inferred the opposite meaning, so let's give a better name. I don't think it needs to be short. `DEFER_TO_IMPLEMENTERS`?

Reply:
I agree we should come up with a better name. I don't think `DEFER_TO_IMPLEMENTERS` reflects the intent, though. There are cases where we actually _do_ want to defer to implementers (e.g., software component architecture, optimization, certain types of market-making), but this is not one of those cases; rather, here we're just saying that while we do intend to implement this logic in the spec eventually, it's unambiguous and will not block implementation if it's not yet written in full detail, and is therefore of lower priority.